docker/artifacts/Dockerfile

```
@@ -1,8 +1,25 @@
 FROM ubuntu:20.04
 
-RUN apt-get update && apt-get install -y python3 python3-pip
-COPY constraints.txt constraints.txt
-ENV PIP_CONSTRAINT=/constraints.txt
+RUN apt-get update && apt-get install -y python3 python3-pip curl gzip
+
+# Install argo for cronjob submission
+# https://github.com/argoproj/argo-workflows/releases/tag/v3.3.1
+RUN curl -sLO https://github.com/argoproj/argo-workflows/releases/download/v3.3.1/argo-linux-amd64.gz && \
+    gunzip argo-linux-amd64.gz && \
+    chmod +x argo-linux-amd64 && \
+    mv argo-linux-amd64 /usr/local/bin/argo && \
+    argo version
```

I love go btw...so nice that this works. We can make it a bit slicker using pipes

```suggestion
RUN curl -sL https://github.com/argoproj/argo-workflows/releases/download/v3.3.1/argo-linux-amd64.gz \
    | gunzip  > /usr/local/bin/argo \
    && chmod +x /usr/local/bin/argo \
    && echo
```